packages/eslint-config-hypothesis/package.json

```diff
@@ -10,16 +10,47 @@
   "scripts": {
     "test": "echo \"Warning: no test specified\" && exit 0"
   },
+  "exports": {
+    "./base": {
+      "import": "./base.js"
+    },
+    "./jsx": {
+      "import": "./jsx.js"
+    },
+    "./ts": {
+      "import": "./ts.js"
+    }
+  },
```

Comment:
I'm adding these exports so that it is possible to import without the `.js` extension.